### PR TITLE
[py] minor

### PR DESCRIPTION
**Pull request by ebraminio**

### CODE REVIEW

File: src/check-c-linkage-decls.py [hunk 1]
```diff
@@ -23,4 +23,4 @@
 		print ('Ouch, file %s has HB_BEGIN_DECLS / HB_END_DECLS, but it shouldn\'t' % x)
 		stat = 1
 
-sys.exit (stat)
+exit (stat)
```
Comment (khaledhosny) [inline src/check-c-linkage-decls.py:26]: https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python

Comment (ebraminio) [reply src/check-c-linkage-decls.py:26]: oh, so let's normalize in the reverse way

**Comment by ebraminio** [comment]
@khaledhosny, is the rest okay? Thank you

**Comment by ebraminio** [comment]
Ok merged as it seems to work, please let me know any other thing with the changes and will try my best to fix them. Thank you so much :-)

**Pull request merged**